src/plugins/presentation_util/public/components/input_controls/control_types/options_list/options_list.scss

```diff
@@ -41,31 +16,32 @@
   max-width: 100%;
 }
 
-.optionsList--control {
-  height: $euiFormControlHeight;
-  flex-direction: row;
-  background-color: $euiFormBackgroundColor;
-  width: 100%;
-  flex-grow: 1;
-  display: flex;
-  align-items: center;
-
-  .optionsList--selections {
-    flex: 1;
+.optionsList--filterBtn {
+  .euiFilterButton__text-hasNotification {
+    flex-grow: 1;
+    justify-content: space-between;
     width: 0;
```

Comment:
Having a `min-width` somewhere is also pretty important I think. Not sure how to handle this, to be honest. Maybe the control frame should have a min-width? With no min width set, it can get pretty squished, and slightly unusable. How do you think we should handle that?

<img width="773" alt="Screen Shot 2021-07-30 at 12 40 21 PM" src="https://user-images.githubusercontent.com/14276393/127677480-0915289d-1cc1-4d0f-a709-694800c99a0d.png">

Reply:
Yeah! I noticed that yesterday too. Not 100% of what's the best route but let's go with setting it on `.controlFrame--formControlLayout` for now